### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Some suggestions for improvements, the "DNS over HTTPS" is the one I noticed the most.
Don't agree to the CoC.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -36,63 +36,63 @@
...
-    <string name="substratum_builtin_module">Substratum builtin module</string>
-    <string name="force_dark_terminal_title">Force dark mode terminal</string>
-    <string name="file_picker_failure">Your current file picker failed to give access to the file.</string>
+    <string name="magisk_builtin_module">Magisk built-in module</string>
+    <string name="substratum_builtin_module">Substratum built-in module</string>
+    <string name="force_dark_terminal_title">Force dark-mode terminal</string>
+    <string name="file_picker_failure">Your current file picker failed to grant access to the file.</string>
     <string name="remote_install_title">Remote install</string>
     <string name="file_picker_wierd">Your file picker returned a non standard response.</string>
     <string name="use_magisk_install_command_pref">Use magisk module install command</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:67]:
```suggestion
    <string name="use_magisk_install_command_pref">Use \"magisk module install\" command</string>
```
?

**Review by Fox2Code — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -36,63 +36,63 @@
...
-    <string name="magisk_builtin_module">Magisk builtin module</string>
-    <string name="substratum_builtin_module">Substratum builtin module</string>
-    <string name="force_dark_terminal_title">Force dark mode terminal</string>
-    <string name="file_picker_failure">Your current file picker failed to give access to the file.</string>
+    <string name="magisk_builtin_module">Magisk built-in module</string>
+    <string name="substratum_builtin_module">Substratum built-in module</string>
+    <string name="force_dark_terminal_title">Force dark-mode terminal</string>
+    <string name="file_picker_failure">Your current file picker failed to grant access to the file.</string>
     <string name="remote_install_title">Remote install</string>
     <string name="file_picker_wierd">Your file picker returned a non standard response.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:66]:
```suggestion
    <string name="file_picker_wierd">Your file picker returned a non-standard response.</string>
```

**Comment by Fox2Code** [comment]
Everything else seems fine to me.
I feel weird for the "from carrying out actions" in lockdown mode description, but since I can't figure out why I think it's maybe just a personal preferences.

**Comment by comradekingu** [comment]
@Fox2Code 
Happy to change it back if you find something you like more.
Hopefully it all can be put on a Weblate instance now that it is more polished.
Not so easy to keep that updated between files otherwise.

**Comment by Fox2Code** [comment]
I'll merge this at the same time as #107 for `0.5.0`. 
Someone reported an optimization bug on telegram 10 minutes after I released `0.4.4`...
So I may do a `0.4.5`, or I may jump to `0.5.0` directly, depend on many things.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+    <string name="failed_download">Could not download the file</string>
     <string name="slow_modules">Modules took too long to boot, consider disabling some modules</string>
-    <string name="fail_internet">Failed to connect to the internet</string>
-    <string name="no_web_view">Failed to get system WebView</string>
+    <string name="fail_internet">Could not connect to the Internet</string>
+    <string name="no_web_view">Could not open system WebView</string>
     <string name="title_activity_settings">SettingsActivity</string>
-    <string name="app_update_available">Application update available</string>
+    <string name="app_update_available">A new version of the app is available</string>
     <string name="app_update">Update</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:16]:
```suggestion
    <string name="app_update">Upgrade</string>
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -26,7 +26,7 @@
     <string name="require_android_6">Require Android 6.0+</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:26]:
```suggestion
    <string name="require_android_6">Requires Android 6.0+</string>
```
?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -36,63 +36,63 @@
...
-    <string name="showcase_mode_desc">Lockdown mode prevent manager to do action on modules</string>
+    <string name="showcase_mode_desc">Lockdown mode prevents the manager from carrying out actions on modules</string>
     <string name="pref_category_settings">Settings</string>
     <string name="pref_category_info">Info</string>
     <string name="show_licenses">Show licenses</string>
     <string name="licenses">Licences</string>
     <string name="show_incompatible_pref">Show incompatible modules</string>
-    <string name="show_incompatible_desc">Show modules that are incompatible with your device based on their metadata</string>
+    <string name="show_incompatible_desc">Show modules unlikely to work on your device based on their metadata</string>
     <string name="magisk_outdated">Magisk is outdated!</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:46]:
```suggestion
    <string name="magisk_outdated">There is a new version of Magisk to install!</string>
```

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -36,63 +36,63 @@
...
-    <string name="master_delete_fail">Failed to delete the module files</string>
+    <string name="master_delete_no">Keep</string>
+    <string name="master_delete_yes">Delete</string>
+    <string name="master_delete_fail">Could not delete the module files</string>
     <string name="theme_pref">Theme</string>
     <string name="theme_mode_pref">Theme mode</string>
-    <string name="module_id_prefix">Module id: </string>
+    <string name="module_id_prefix">Module ID: </string>
     <string name="install_from_storage">Install module from storage</string>
     <string name="invalid_format">The selected module is in an invalid format</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:58]:
```suggestion
    <string name="invalid_format">The selected module has an invalid format</string>
```
This is minute, but I think if it is the file extention "has" is better, and
if the internal anything is the problem then "is in" is better.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -36,63 +36,63 @@
...
     <string name="local_install_title">Local install</string>
     <string name="source_code">Source code</string>
-    <string name="magisk_builtin_module">Magisk builtin module</string>
-    <string name="substratum_builtin_module">Substratum builtin module</string>
-    <string name="force_dark_terminal_title">Force dark mode terminal</string>
-    <string name="file_picker_failure">Your current file picker failed to give access to the file.</string>
+    <string name="magisk_builtin_module">Magisk built-in module</string>
+    <string name="substratum_builtin_module">Substratum built-in module</string>
+    <string name="force_dark_terminal_title">Force dark-mode terminal</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:63]:
```suggestion
    <string name="force_dark_terminal_title">Dark terminal</string>
```
?

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -36,63 +36,63 @@
...
+    <string name="use_magisk_install_command_pref">Use the \"magisk --install-module\" command</string>
     <string name="use_magisk_install_command_desc">
-        During test it caused problems to the module install error diagnosis tool,
-        so I hid this option behind developer mode, enable this at your own risk!
+        During testing it caused problems to the module install error diagnosis tool,
+        so this option behind is hidden behind developer mode. Turn this on at your own risk!
     </string>
-    <string name="dev_mode_enabled">Developer mode enabled</string>
+    <string name="dev_mode_enabled">Developer mode on</string>
     <string name="force_english_pref">Force English language</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:73]:
```suggestion
    <string name="force_english_pref">English app language</string>
```

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -36,63 +36,63 @@
...
     <string name="local_install_title">Local install</string>
     <string name="source_code">Source code</string>
-    <string name="magisk_builtin_module">Magisk builtin module</string>
-    <string name="substratum_builtin_module">Substratum builtin module</string>
-    <string name="force_dark_terminal_title">Force dark mode terminal</string>
-    <string name="file_picker_failure">Your current file picker failed to give access to the file.</string>
+    <string name="magisk_builtin_module">Magisk built-in module</string>
+    <string name="substratum_builtin_module">Substratum built-in module</string>
+    <string name="force_dark_terminal_title">Force dark-mode terminal</string>
+    <string name="file_picker_failure">Your current file picker failed to grant access to the file.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:64]:
```suggestion
    <string name="file_picker_failure">Your current file picker could not access the file.</string>
```
Same same? Is there a reason why it isn't a file browser?

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -36,63 +36,63 @@
...
-        During test it caused problems to the module install error diagnosis tool,
-        so I hid this option behind developer mode, enable this at your own risk!
+        During testing it caused problems to the module install error diagnosis tool,
+        so this option behind is hidden behind developer mode. Turn this on at your own risk!
     </string>
-    <string name="dev_mode_enabled">Developer mode enabled</string>
+    <string name="dev_mode_enabled">Developer mode on</string>
     <string name="force_english_pref">Force English language</string>
-    <string name="disable_low_quality_module_filter_pref">Disable low quality module filter</string>
+    <string name="disable_low_quality_module_filter_pref">Turn off low quality module filter</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:74]:
```suggestion
    <string name="disable_low_quality_module_filter_pref">Show low quality modules</string>
```
?

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -36,63 +36,63 @@
...
     <string name="disable_low_quality_module_filter_desc">
-        Some modules do not declare their metadata properly,causing visual glitches,
-        and/or indicating poor module quality, disable at your own risk!
+        Some modules do not declare their metadata properly, causing visual glitches,
+        and/or indicating poor module quality. Turn this off at your own risk!
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:77]:
```suggestion
        and/or indicating poor module quality.\nTurn this off at your own risk!
```
?

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -36,63 +36,63 @@
...
-        (Does not apply to WebView)
+        (Does not apply to WebView.)
     </string>
-    <string name="disable_extensions_pref">Disable extensions</string>
+    <string name="disable_extensions_pref">No Mmm</string>
     <string name="disable_extensions_desc">
-        Disable Fox\'s Mmm extensions, this prevent modules from using
-        terminal extensions, useful if a module misuse Fox\'s Mmm extensions.
+        Turn off Fox\'s Mmm extensions, preventing modules from using
+        terminal extensions. Useful if a module misuses Fox\'s Mmm extensions.
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:87]:
```suggestion
        terminal extensions.\nUseful if a module misuses Fox\'s Mmm extensions.
```
?

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -36,63 +36,63 @@
...
     </string>
     <string name="wrap_text_pref">Wrap text</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:89]:
```suggestion
    <string name="wrap_text_pref">Text wrapping</string>
```
Slightly better?

File: app/src/main/res/values/strings.xml [hunk 14]
```diff
@@ -36,63 +36,63 @@
...
     <string name="wrap_text_desc">
         Wrap text to a new line instead of putting
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:91]:
```suggestion
        Show text on multiple lines instead of putting
```

**Comment by comradekingu** [comment]
@Fox2Code After making the nb_NO translations I found some improvements, tried to avoid inconsistencies, and added whatever small improvements I could find.
If you find the time, maybe you can have a look. Nothing showstopping.

**Comment by Fox2Code** [comment]
Yeah everything you did seems fine.
Also for context `disable_low_quality_module_filter_pref` is disabled in release builds.
Also you not agreeing to CoC doesn't means it doesn't apply here, see #135
I am the only one that need to agree to it as I'm the one handling bans, but I am not strict in enforcing it.

**Comment by Fox2Code** [comment]
I have reworked code of conduct, you can tell me what you think.
Also you didn't expressed if you wanted or not to be credited or not.
I will assume you don't want to receive credits again, but I would still like you to confirm that if possible.

**Pull request merged**